client/styles/components/_forms.scss

```diff
@@ -79,6 +84,10 @@
   top: 7px;
   right: -355px;
```

Comment:
My comment is related moreso to the existing code than to your PR.  This sort of positioning based on hard-coded pixel values is extremely fragile.  355 pixels away from what?  @raclim we should revisit this.  Put both the eye `button` and the `input` inside of a `div` with `position: relative`.  Then use `position: absolute; right: 0` on the `button` to have it be aligned to the right side of the input.  If we do that then we shouldn't need this new media query to adjust the `right` position because it would always be `0` regardless of screen size.

Reply:
I think that completely works. Maybe we could make those adjustments within this PR if @rushikeshg25 is down?